MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSAppleErrorLog.m

```diff
@@ -86,6 +86,9 @@ - (BOOL)isValid {
 - (BOOL)isEqual:(MSAppleErrorLog *)errorLog {
   if (!errorLog)
     return NO;
+  
+  if (![super isEqual:errorLog])
```

Comment:
@MatkovIvan I already merged this but thx to @jaelim-ms  I have a question for you:
what's the reason for this addition?!